apps/saleor-app-checkout/backend/configuration/settings.ts

```diff
@@ -47,36 +56,46 @@ export const getPrivateSettings = async (apiUrl: string, obfuscateEncryptedData:
   return settingsValues;
 };
 
-export const getPublicSettings = async () => {
-  const { data, error } = await getClient()
+export const getPublicSettings = async (domain: string) => {
+  const authData = await apl.get(domain);
```

Comment:
Does the APL cache it somehow?

Reply:
Not implemented. Memcache could be enough for a request lifetime 🤔